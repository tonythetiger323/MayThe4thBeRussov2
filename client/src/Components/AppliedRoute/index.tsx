// Import dependencies
import * as React from 'react'
import { Route } from 'react-router-dom'

// Creating a route that will apply child props
const AppliedRoute = ({ component: C, props: cProps, ...rest }: any) => {
  return (
    <Route {...rest} render={(props: any) => <C {...props} {...cProps} />} />
  )
}
export default AppliedRoute
