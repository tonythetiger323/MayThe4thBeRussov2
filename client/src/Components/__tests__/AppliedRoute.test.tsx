import * as React from 'react'
import { shallow } from 'enzyme'
import AppliedRoute from '../AppliedRoute'

describe('AppliedRoute Component', () => {
  const appliedRoute = shallow(<AppliedRoute />)
  it('should render correctly', () => {
    expect(appliedRoute).toMatchSnapshot()
  })
})
