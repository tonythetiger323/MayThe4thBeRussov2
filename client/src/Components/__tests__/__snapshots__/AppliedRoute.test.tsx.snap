// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppliedRoute Component should render correctly 1`] = `ShallowWrapper {}`;
